support-frontend/assets/pages/[countryGroupId]/components/thankYouComponent.tsx

```diff
@@ -178,20 +203,32 @@ export function ThankYouComponent({
 		contributionType,
 		currencyKey,
 	);
-
-	const printProductsKeys: ActiveProductKey[] = [
+	const subscriptionCardProductsKey: ActiveProductKey[] = ['SubscriptionCard'];
+	const paperProductsKeys: ActiveProductKey[] = [
 		'NationalDelivery',
 		'HomeDelivery',
-		'SubscriptionCard',
+	];
+	const printProductsKeys: ActiveProductKey[] = [
+		...subscriptionCardProductsKey,
+		...paperProductsKeys,
 		'GuardianWeeklyDomestic',
 		'GuardianWeeklyRestOfWorld',
 	];
 	const isPrint = printProductsKeys.includes(productKey);
+	const observerPrint: ObserverPrint | undefined =
+		paperProductsKeys.includes(productKey) && ratePlanKey === 'Sunday'
+			? 'ObserverPaper'
+			: subscriptionCardProductsKey.includes(productKey) &&
+			  ratePlanKey === 'Sunday'
+			? 'ObserverSubscriptionCard'
+			: undefined;
```

Comment:
one more for the nested ternaries
```suggestion
	const observerPrint: ObserverPrint | null =
		if (paperProductsKeys.includes(productKey) && ratePlanKey === 'Sunday') {
		  return 'ObserverPaper';
		}
		
		if (subscriptionCardProductsKey.includes(productKey) && ratePlanKey === 'Sunday') {
		  return 'ObserverSubscriptionCard';
		}
		
                return null;
      }
```